Ventivader/Ventivader/Utilities/LoadingOverlay.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  LoadingOverlay.swift
+//  Ventivader
+//
+//  Created by Al on 19/04/20.
+//  Copyright © 2020 calculator. All rights reserved.
+//
+import UIKit
+import Foundation
+import SwiftGifOrigin
+
+
+public class LoadingOverlay {
+    
+    var overlayView = UIView()
+    //var activityIndicator = UIActivityIndicatorView()
+    
+    private lazy var bleLoaderView: UIView = {
+        let containerView = UIView()
+        containerView.backgroundColor = .white
+        
+        let animationView = UIImageView()
+        animationView.loadGif(name: "bleConnectingAnimation")
+        
+        let titleLabel = UILabel()
```

Comment:
would be good to later encapulate this code into a private method  for readbility.